src/main/scala/com/greencatsoft/angularjs/internal/Angular.scala

```diff
@@ -12,6 +14,8 @@ import com.greencatsoft.angularjs.Filter
 @js.native
 private[angularjs] trait Angular extends js.Object {
 
+  def injector(): Injector = js.native
```

Comment:
Can we get away with that? Perhaps I'm confusing myself but I think you have to call js methods with parens for the function to get invoked. Perhaps I'm being silly, otherwise I agree.